skmultilearn/tree/pctcriterium.pyx

```diff
@@ -0,0 +1,638 @@
+# cython: cdivision=True
```

Comment:
When generating the C sources from this file with Cython==0.28.2 and python 3.6.5 (`cython -3 -o pctcriterium.c pctcriterium.pyx`), there are many errors - most frequently about operations on python variables without GIL being not allowed.